src/schemas/components/SchemaFormPropertyAnyOfInput.vue

```diff
@@ -1,12 +1,80 @@
 <template>
-  <span>TODO</span>
+  <div class="schema-form-property-any-of-input">
+    <p-button-group v-model="selectedPropertyIndex" :options="options" small />
+  </div>
+
+  <p-card>
+    <SchemaFormProperty :key="selectedPropertyIndexValue" v-model:value="value" :property="property" :required="required" />
+  </p-card>
 </template>
 
 <script lang="ts" setup>
-  import { SchemaProperty } from '@/schemas/types/schema'
+  import { ButtonGroupOption } from '@prefecthq/prefect-design'
+  import { computed, reactive, ref } from 'vue'
+  import SchemaFormProperty from '@/schemas/components/SchemaFormProperty.vue'
+  import { useSchema } from '@/schemas/compositions/useSchema'
+  import { SchemaProperty, isPropertyWith } from '@/schemas/types/schema'
+  import { SchemaValue } from '@/schemas/types/schemaValues'
+  import { getSchemaDefinition } from '@/schemas/utilities/definitions'
   import { Require } from '@/types/utilities'
 
-  defineProps<{
+  const props = defineProps<{
     property: Require<SchemaProperty, 'anyOf'>,
+    value: SchemaValue,
+    required: boolean,
+  }>()
+
+  const emit = defineEmits<{
+    'update:value': [SchemaValue],
   }>()
+
+  const value = computed({
+    get() {
+      return propertyValues[selectedPropertyIndex.value]
```

Comment:
This might be a can of worms but is the order of `anyOf` property definitions guaranteed? Could the order change between requests?

Reply:
I don't think we should consider the order to be guaranteed. But it shouldn't matter here. Once we set the initial index things will stay in sync as the user switches between definitions.